entitlement/src/main/resources/org/killbill/billing/entitlement/dao/BlockingStateSqlDao.sql.stg

```diff
@@ -104,7 +104,35 @@ where <accountRecordIdField("t.")> = :accountRecordId
 <AND_CHECK_TENANT("t.")>
 <defaultOrderBy("t.")>
 ;
- >>
+>>
+
+
+getBlockingActiveForAccount() ::= <<
+select
+<allTableFields("bs1.")>
+from
+blocking_states bs1
+join (
+  select
+   service
+  , blockable_id
+  , count(*)
+  from
+  blocking_states
+  where 1=1
+  and <accountRecordIdField("")> = :accountRecordId
+  <AND_CHECK_TENANT("")>
+  and block_billing
```

Comment:
You might want to specify `and block_billing = '1'` so that it's standardised across engines -- to be checked how we do it elsewhere (e.g. `subscription_events.is_active`).

Reply:
See https://github.com/killbill/killbill/pull/1566/commits/289758ba807b0c4d3ba811130176bb7a96a050a2